gum/gummemory.h

```diff
@@ -88,6 +88,8 @@ gpointer gum_alloc_n_pages (guint n_pages, GumPageProtection page_prot);
 gpointer gum_alloc_n_pages_near (guint n_pages, GumPageProtection page_prot, GumAddressSpec * address_spec);
 void gum_free_pages (gpointer mem);
 
+gint gum_page_protection_to_posix (GumPageProtection page_prot);
```

Comment:
This header file is public and available on all OSes; we need to introduce an internal header file. Remember to prefix the function with an underscore and mark it G_GNUC_INTERNAL.